cmd/utils/flags.go

```diff
@@ -2187,6 +2204,16 @@ func MakeGenesis(ctx *cli.Context) *core.Genesis {
 		genesis = core.DefaultRinkebyGenesisBlock()
 	case ctx.Bool(GoerliFlag.Name):
 		genesis = core.DefaultGoerliGenesisBlock()
+	case ctx.IsSet(BetaOPNetworkFlag.Name):
+		ch, err := params.OPStackChainIDByName(ctx.String(BetaOPNetworkFlag.Name))
+		if err != nil {
+			Fatalf("failed to load OP-Stack chain: %v", err)
+		}
+		genesis, err := core.LoadOPStackGenesis(ch)
+		if err != nil {
+			Fatalf("failed to load genesis for OP-Stack chain %d: %v", ch, err)
```

Comment:
Can we include the network name in this error message? It would be very confusing to say `--beta.op-network=foo` and then get the error `failed to load genesis for OP-Stack chain 502`.  It shouldn't ever happen given the chain ID lookup is based on the same list of chain configs but still...

Reply:
Good point, will update this